conformance/fhir-ig-us-core/src/main/java/com/ibm/fhir/ig/us/core/util/IndexGenerator.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.ig.us.core.util;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.OutputStream;
+
+import com.ibm.fhir.model.format.Format;
+import com.ibm.fhir.model.parser.FHIRParser;
+import com.ibm.fhir.model.parser.exception.FHIRParserException;
+import com.ibm.fhir.model.resource.Resource;
+import com.ibm.fhir.registry.util.Index;
+import com.ibm.fhir.registry.util.Index.Entry;
+
+public class IndexGenerator {
```

Comment:
did we need to generate our own `.index.json` file for US Core?  does it include an `.index.json` that just isn't valid or does it not even have one?

Reply:
This is a net new file. The version in master does not have an IndexGenerator utility class. @prb112 does the latest. version not include a `.index.json` file?

Reply:
so - I totally own this one... I copied from a directory (. is hidden) so I had to generate the index.  I'll copy over the official one and remove the Generator.  This is on me.

Reply:
Fixed (lesson learned on hidden files)